spec/models/course/level_spec.rb

```diff
@@ -2,11 +2,55 @@
 
 RSpec.describe Course::Level, type: :model do
   it { is_expected.to belong_to(:course).inverse_of(:levels) }
-  it { is_expected.to validate_numericality_of(:experience_points_threshold).is_greater_than(0) }
+  it 'ensures that experience points threshold is greater or equal to 0' do
+    expect(subject).
+      to validate_numericality_of(:experience_points_threshold).
+      is_greater_than_or_equal_to(0)
+  end
+
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    let!(:course) { build(:course) }
+
+    context 'when new level have the same threshold as existing level' do
```

Comment:
Materials folders did it, I'd very much prefer to have this, would recommend a validations block. 

It will be easier to refactor (if one day we need to change how default levels will be implemented).